src/main/kotlin/com/openlattice/postgres/PostgresDataTables.kt

```diff
@@ -142,6 +143,8 @@ class PostgresDataTables {
                     .name(prefix+"_read_data_idx")
                     .ifNotExists()
 
+            val needsPropagateIndex = PostgresExpressionIndexDefinition(tableDefinition, "(${LAST_WRITE.name} > ${LAST_PROPAGATE.name})")
```

Comment:
Is last_propagate being used to mean that a row needs to be (re)materialized now?

Reply:
Yes.

Reply:
Are we going to introduce a new column for old propagation once graph queries are fixed (ie auto-setting data values based on neighbor data values, etc) or are we just scrapping that feature?

Reply:
We can add a new column later when we get to graph propagation.

Reply:
Might be worth renaming this column then? To avoid confusion with graph propagation

Reply:
To like last_materialize or last_assemble or last_transport or something

Reply:
The more I think about it the more I think a new column is safer.